app/services/search/tag.rb

```diff
@@ -75,6 +75,9 @@ def mappings
             },
             short_summary: {
               type: "text"
+            },
+            rules_html: {
```

Comment:
Discovered upon hooking up search that we need to return this attribute as sometimes [it is included in the dropdown](https://github.com/thepracticaldev/dev.to/blob/a86ccff096c9f3a142ce903d476662ff64f0f05e/app/javascript/shared/components/tags.jsx#L393) when searching.

Reply:
Good catch 🧐